exercises/practice/affine-cipher/.docs/instructions.md

```diff
@@ -0,0 +1,74 @@
+# Instructions
+
+Create an implementation of the affine cipher, an ancient encryption system created in the Middle East.
+
+The affine cipher is a type of monoalphabetic substitution cipher.
+Each character is mapped to its numeric equivalent, encrypted with a mathematical function and then converted to the letter relating to its new numeric value.
+Although all monoalphabetic ciphers are weak, the affine cipher is much stronger than the atbash cipher, because it has many more keys.
+
+[//]: # " monoalphabetic as spelled by Merriam-Webster, compare to polyalphabetic "
+
+## Encryption
+
+The encryption function is:
+
+```text
+E(x) = (ai + b) mod m
```

Comment:
From mathematical notation this is weird. `E(x)` means that this is a function called `E` that depends on the variable `x`. But there is no `x` in the calculation.

I think it should be `E(i)` or `i` changed to `x` in the function.

Reply:
This is something that would need to be fixed upstream, in the prob-specs repo.